Dockerfile

```diff
@@ -28,6 +29,30 @@ RUN ldconfig
 
 WORKDIR /home/V2X-Hub/
 RUN /home/V2X-Hub/container/setup.sh
+
+# run final image
+FROM ubuntu:$UBUNTU_VERSION AS v2xhub
+ENV DEBIAN_FRONTEND=noninteractive
+ADD scripts/install_dependencies.sh /usr/local/bin/
```

Comment:
I am not sure I understand this at all. If you are trying to setup build stages, my understanding is that you would install dependencies in the first build stage and only copy over the installed dependencies. Not sure how this makes sense if you install dependencies in both images. It looks like most of the saved memory is from just not including V2X-Hub core source files

Reply:
Most of the dependencies in the script are still used. I think cmake and git wouldn't be needed anymore, but the rest look like they're still used

Reply:
addressed

Reply:
It still looks like we are running the install dependencies twice. Am I mistaken?